dev/com.ibm.ws.microprofile.faulttolerance.3.0_fat_tck/publish/tckRunner/tck/tck-suite-lite.xml

```diff
@@ -0,0 +1,57 @@
+<!-- Licensed under the Apache License, Version 2.0 (the "License"); you 
+    may not use this file except in compliance with the License. You may obtain 
+    a copy of the License at http://www.apache.org/licenses/LICENSE-2.0 Unless 
+    required by applicable law or agreed to in writing, software distributed 
+    under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES 
+    OR CONDITIONS OF ANY KIND, either express or implied. See the License for 
+    the specific language governing permissions and limitations under the License. -->
+<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd" >
+
+<suite name="microprofile-faulttolerance-2.1-TCK" verbose="2"
+    configfailurepolicy="continue">
+
+    <test name="microprofile-faulttolerance 2.1 TCK">
+        <method-selectors>
+            <method-selector>
+                <script language="beanshell">
+                     <!-- Some tests are sensitive to timing and can spuriously fail on slow build machines. -->
+                     <!-- We exclude all of these tests when running in lite mode -->
+                     <![CDATA[
```

Comment:
Are we going to fix the timing on these tests for 3.0?

Reply:
I hope so, we have https://github.com/eclipse/microprofile-fault-tolerance/issues/271 open and assigned to the 3.0 milestone